src/h5web/providers/utils.ts

```diff
@@ -108,3 +108,7 @@ export async function handleAxiosError<T>(
     throw error;
   }
 }
+
+export function encodeQueryParams(params: Record<string, string>) {
+  return new URLSearchParams(params).toString();
```

Comment:
I used `URLSearchParams` as it was easier for what I wanted to do. There was other ways of doing this and here why I discarded them:
- [qs.stringify](https://github.com/ljharb/qs#stringifying): adds a new dependency
- `encodeURIComponent`: require looping by hand (see https://stackoverflow.com/questions/59889140/different-output-from-encodeuricomponent-vs-urlsearchparams for an example implementation)
- pass query params with the `params` field of the `AxiosConfig`: tempting but require additional changes to `cancellableFetchValue`

Reply:
Great call 💯